index.html

```diff
@@ -1504,6 +1504,47 @@ <h2>Public Keys</h2>
 see Section <a href="#resolution"></a>.
       </p>
 
+<p>
+Revoked keys MUST NOT be present in the <code>publicKey</code> property of a <a>DID document</a>.<br/>
+Revoked keys MAY be present in the <code>revokedKey</code> property of a <a>DID document</a>.
+</p>
+
+<pre class="example nohighlight" title="Revoked public keys">
+{
+  "@context": ["https://www.w3.org/ns/did/v1", "https://w3id.org/security/v1"],
+  "id": "did:example:123456789abcdefghi",
+  <span class="comment">...</span>
+  "publicKey": [],
+  "revokedKey": [{
+    "id": "did:example:123#_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A",
+    "type": "JwsVerificationKey2020",
+    "controller": "did:example:123",
+    "publicKeyJwk": {
+      "crv": "Ed25519",
+      "x": "VCpo2LMLhn6iWku8MKvSLg2ZAoC-nlOyPVQaO3FxVeQ",
+      "kty": "OKP",
+      "kid": "_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A"
+    },
+    "revocation": {
+      <span class="comment">...</span>
```

Comment:
It's not clear to me what goes in side the "revocation" tag. Can we expand the example a little bit more than just "..."?

Reply:
its not clear to me either :) 

The spec language todays, says it can be embedded or a remote revocation.... But I have never seen an example of either.

I could include GPG Revocation Certificate and an example using https://gpg.jsld.org/

See also: https://debian-administration.org/article/450/Generating_a_revocation_certificate_with_gpg

But as the author of that signature suite, I'm pretty sure nobody other than me is using it right now :)

If there is no better example, I can probably update the GPG Signature suite to provide a revocation certificate example in both embedded and remote form.

Reply:
As I have revised this content, I request to resolve this thread to avoid confustion.